jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -67,11 +78,80 @@
 
   @CommandLine.Option(
       names = "--from",
-      paramLabel = "<from>",
+      paramLabel = "<base-image>",
       description = "The base image to use.")
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private String from;
 
+  @CommandLine.Option(
+      names = "--jvm-flags",
+      paramLabel = "<jvm-flags>",
+      split = ",",
+      description = "JVM arguments, example: --jvm-flags=-Dmy.property=value,-Xshare:off")
+  private List<String> jvmFlags = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--expose",
+      paramLabel = "<exposed-ports>",
+      split = ",",
+      description = "Ports to expose on container, example: --expose=5000,7/udp.")
+  private List<String> exposedPorts = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--volumes",
+      paramLabel = "<volumes>",
+      split = ",",
+      description =
+          "Directories on container to hold extra volumes,  example: --volumes=/var/log,/var/log2.")
+  private List<String> volumes = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--environment-variables",
+      paramLabel = "<environment-variables>",
+      split = ",",
+      description =
+          "Environment variables to write into container, example: --environment-variables env1=env_value1,env2=env_value2.")
+  private Map<String, String> environment = new LinkedHashMap<>();
+
+  @CommandLine.Option(
+      names = "--labels",
+      paramLabel = "<labels>",
+      split = ",",
+      description =
+          "Labels to write into container metadata, example: --labels=label1=value1,label2=value2.")
+  private Map<String, String> labels = new LinkedHashMap<>();
+
+  @CommandLine.Option(
+      names = {"-u", "--user"},
+      paramLabel = "<user>",
+      description = "The user to run the container as, example: --user=myuser:mygroup.")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private String user;
+
+  @CommandLine.Option(
+      names = {"--image-format"},
+      defaultValue = "Docker",
+      paramLabel = "<image-format>",
+      description =
+          "Format of container, candidates: ${COMPLETION-CANDIDATES}, default: ${DEFAULT-VALUE}.")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private ImageFormat format;
+
+  @CommandLine.Option(
+      names = "--program-args",
+      paramLabel = "<program-arguments>",
+      split = ",",
+      description = "Program arguments for container entrypoint.")
+  private List<String> programArguments = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--entrypoint",
+      paramLabel = "<entrypoint>",
+      split = "\\s+",
```

Comment:
Q: does picoCLI automatically takes care of escaping for the split characters? For example, for `--program-args`, is  there a way to include `,` in an argument?

Reply:
When using split regex with picocli I think the user still needs to take care of escaping. A backlash needs to be escaped with another backlash otherwise it won't compile. In the case of `--program-args`, there is a way to include `,` in an argument. For example `--program-args=\"Hello, World\", arg2` will result in `["Hello,World", arg2]`. Does this answer your question?

Reply:
Basically yes. As long as there is a way to pass one of the delimiters, we should be good.